test/gui/tst_sharing/test.feature

```diff
@@ -301,3 +301,16 @@ Feature: Sharing
             | edit        | read, share                  | read, share              |
             | share       | read, update, create, delete | read,update              |
             | edit,share  | read                         | read                     |
+
+
+    Scenario: sharee deletes a file shared by sharer
+        Given the setting "shareapi_auto_accept_share" on the server of app "core" has been set to "yes"
+        And the administrator on the server has set the default folder for received shares to "Shares"
+        And user "Alice" has uploaded file with content "ownCloud test text file 0" to "textfile.txt" on the server
+        And user "Brian" has been created on the server with default attributes and without skeleton files
+        And user "Alice" has shared file "textfile.txt" on the server with user "Brian" with "all" permissions
+        And user "Brian" has set up a client with default settings
+        When the user deletes the file "Shares/textfile.txt"
+        And the user waits for the files to sync
```

Comment:
related to above comment
```suggestion
        And the user waits for file "Shares/textfile.txt" to be synced
```

Reply:
See if it works.

Reply:
This step won't work here as we have deleted file `Shares/textfile.txt` in line 313 but as you mentioned in the comment above we can use this to check if the file has been synced or not.

Reply:
Ohh, Yeah!!